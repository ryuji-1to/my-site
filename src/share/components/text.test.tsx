import { render } from "@testing-library/react";
import { Text } from "./text";

describe("share/components/text", () => {
  test("snapshot", () => {
    const { getByText } = render(<Text>test</Text>);
    expect(getByText("test")).toMatchSnapshot();
  });
});
