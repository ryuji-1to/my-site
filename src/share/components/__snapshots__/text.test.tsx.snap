// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`share/components/text > snapshot 1`] = `
<p
  class="text-gray-800 dark:text-gray-200"
>
  test
</p>
`;
